### ISSUE TITLE
Editors get logged out of Umbraco 8.18.11 when trying to edit a link in the RTE. Administrators are fine.

### ISSUE THREAD

**Issue by tenshi04** [umbraco/Umbraco-CMS#15467]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

8.18.11

### Bug summary

As an Editor, they try to update an existing link in the Rich Text editor. They click on the link, and then click the link icon in the RTE control bar and they immediately get logged out.

If their permissions get elevated to Administrator, then they do not have this issue.

Version is Umbraco 8.18.11

### Steps to reproduce

1. Login as an Editor
2. Go to an RTE
3. CLick on the Link button in the RTE control bar
4. The user is logged out.

1. Login as an Administrator
2. Go to an RTE
3. CLick on the Link button in the RTE control bar
4. The link panel opens without problem.

### Expected result / actual result

The editor user should not be logged out when editing a link in the RTE

**Comment by andr317c** [comment]
Hey! 
Thanks for reporting this issue. I was unable to reproduce it, I followed the reproduction steps and it works without any issues trying to update a link in an RTE for an editor. Are you able to reproduce this on a clean install?

**Comment by tenshi04** [comment]
I think that I figured it actually might be something else.


  1.  User is logged and is a member of a group that mimics the Editor group, but only has access to a certain content node and only a certain media folder.
  2.  On a content page, in the RTE, they actually have a link that is 'broken' and points to a media item that has been moved to the recycle bin.
  3.  When they try to click on the link in the RTE to try to correct it, the get logged out immediately
  4.  If I am logged in as an administrator (or Editor with no media folder restrictions), I do not get logged out when I try to correct the link. I think this is because I have access to the recycle bin? The group that the user belongs to does not have access to the recycle bin and I believe that this is why this is happening?

Does this make sense? Should they be logged out or should they have the ability to correct the link?



Classification: Protected A
